test/index.spec.ts

```diff
@@ -364,13 +364,14 @@ describe('onBuild()', () => {
     expect(existsSync(`.netlify/functions-internal/___netlify-odb-handler/handlerUtils.js`)).toBeTruthy()
   })
 
-  it('writes correct redirects to netlifyConfig', async () => {
+  it('when splitting API routes is disabled, it writes correct redirects to netlifyConfig', async () => {
```

Comment:
this should set env var to `false`

currently Github Actions will have env var set so this test will actually run `onBuild` with API routes enabled

Reply:
👍🏻

Reply:
Done.